sdk/tables/azure-data-tables/azure/data/tables/aio/_table_service_client_async.py

```diff
@@ -283,10 +290,10 @@ def list_tables(
         """Queries tables under the given account.
 
         :keyword int results_per_page: Number of tables per page in return ItemPaged
-        :keyword Union[str, list(str)] select: Specify desired properties of a table to return certain tables
+        :keyword Union[str,list(str)] select: Specify desired properties of a table to return certain tables
```

Comment:
idt `Union` renders correctly in sphinx, you might have to split this into 2 lines:

```
:keyword select: ...
:paramtype select: str or list[str]
```